src/main.cpp

```diff
@@ -5,6 +5,8 @@
 #include <vector>
 #include <string>
 
+//#include "player.cpp"
+
```

Comment:
Lettuce try to not have commented-out code tho i don't mind so much.

BTW if you change anything after a PR, commit, and push to the same branch, this PR should get updated automagically